lib/library_test_utils.go

```diff
@@ -1099,7 +1099,7 @@ func testDiscardTransaction(t *testing.T) bool { //nolint: gocyclo
 		To:    account.ToAddress(TestConfig.Account2.Address),
 		Value: (*hexutil.Big)(big.NewInt(1000000000000)),
 	})
-	time.Sleep(1 * time.Second)
+	time.Sleep(5 * time.Second)
```

Comment:
could we check this condition five times every second instead of sleeping for 5 seconds? in case its accomplished on first iteration we could save 4 extra seconds.